CLDR-15034 kbd: doc naming updates from Sep 12/13, 2022 (#2360)
unicode-org/cldr: docs/ldml/tr35-keyboards.md

Keyboard use can be challenging for individuals with various types of disabilities. For this revision, the committee is not evaluating features or architectural designs for the purpose of improving accessibility. Such consideration could be fruitful for future revisions. However, some points on this topic should be made:

1. Having an industry-wide standard format for keyboards will enable accessibility software to make use of keyboard data with a reduced dependence on platform-specific knowledge.

> 2. Some features, such as multiTap and flicks, have the potential to reduce accessibility and thus should be discouraged. For example, multiTap requires pressing keys at a certain speed, and flicks require a more complex movement (press-and-flick) beyond a simple tap.

Reviewer: ```suggestion
2. Some features, such as multiTap and flicks, have the potential to reduce accessibility and thus should be discouraged. For example, multiTap requires pressing keys at a certain speed, and flicks require a more complex movement (press-and-flick) beyond a simple tap. Alternatively, inclusion of accessible methods of generating the same outputs (for example, simple keys on an additional layer), should be considered.
```

Author: +1000

* * *

### <a name="Element_displays" href="#Element_displays">Element: displays</a>

The displays can be used to describe what is to be displayed on the keytops for various keys. For the most part, such explicit information is unnecessary since the `@to` element from the `keys/key` element can be used. But there are some characters, such as diacritics, that do not display well on their own and so explicit overrides for such characters can help.
Another useful scenario is where there are doubled diacritics, or multiple characters with spacing issues.

The `displays` consists of a list of display subelements.

> displayss are designed to be shared across many different keyboard layout descriptions, and imported with `<import>` in where needed.

Reviewer: ```suggestion
`displays` elements are designed to be shared across many different keyboard layout descriptions, and imported with `<import>` in where needed.
```

The double-s may have been intentional but very confusing!

Author: no, it was search and replace… (displayMaps)s -> (displays)s

The `displays` consists of a list of display subelements.

> `displays` elements are designed to be shared across many different keyboard layout descriptions, and imported with `<import>` in where needed.

Reviewer: ```suggestion
`displays` elements are designed to be shared across many different keyboard layout descriptions, and imported with `<import>` where needed.
```

Missed that last time...

        <key id="a" to="a" longpress="\u0301 \u0300" />
        <key id="shift" switch="shift" />
    </keys>
    <displays>
        <display to="\u0300" display="ˋ" /> <!-- \u02CB -->
        <display to="\u0301" display="ˊ" /> <!-- \u02CA -->
        <display id="shift"  display="⇪" /> <!-- U+21EA -->
    </displays>
</keyboard>
```

> To allow `displays`s to be shared across keyboards, there is no requirement that `@to` in a `display` element matches any `@to`/`@id` in any `keys/key` element in the keyboard description.

Reviewer: ```suggestion
To allow `displays` elements to be shared across keyboards, there is no requirement that `@to` in a `display` element matches any `@to`/`@id` in any `keys/key` element in the keyboard description.
```

```xml
<keyboard>
    <vkeys>

> 		<vkey from="Q" to="A" />
> 		<vkey from="W" to="Z" />
> 		<vkey from="A" to="Q" />
> 		<vkey from="Z" to="W" />

Reviewer: ```suggestion
		  <vkey from="Q" to="A" />
		  <vkey from="W" to="Z" />
		  <vkey from="A" to="Q" />
		  <vkey from="Z" to="W" />
```

   * The **quaternary weight** is the index of the character in the string. This is solely to ensure a stable sort for sequences of characters with the same tertiary weight.
2. Mark each character as to whether it is a prebase character, one that is typed before the base and logically stored after. Thus it will have a primary order > 0.
3. Use the sort key and the prebase mark to identify runs. A run starts with a prefix that contains any prebase characters and a single base character whose primary and tertiary key is 0. The run extends until, but not including, the start of the prefix of the next run or end of the string.

>    * `run := preBase* (primary=0 && tertiary=0) ((primary≠0 || tertiary≠0) && !prebase)*`

Reviewer: ```suggestion
   * `run := preBase* (primary=0 && tertiary=0) ((primary≠0 || tertiary≠0) && !preBase)*`
```
